elixir/apps/web/lib/web/live/policies/components.ex

```diff
@@ -235,7 +235,12 @@ defmodule Web.Policies.Components do
     ~H"""
     <fieldset class="flex flex-col gap-2 mt-4">
       <div class="flex items-center justify-between">
-        <legend class="text-xl mb-2 text-neutral-900">Conditions</legend>
+        <div>
+          <legend class="text-xl mb-2 text-neutral-900">Conditions</legend>
+          <p class="my-2 text-sm text-neutral-500">
+            When specified, all conditions below must be met for this policy to be applied.
```

Comment:
English is so ambiguous at times 😄 
Do you think someone could interpret this as "if any are specified then all must be met..."?

Reply:
Yeah, but that's the intention. Maybe `If specified, ...`?

Reply:
What do you think about "All specified conditions below..." ?
Honestly, I don't have strong feelings, just a quick gut check is all.